compiler/qsc_rca/src/scaffolding.rs

```diff
@@ -0,0 +1,296 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::{
+    common::{GlobalSpecId, SpecKind},
+    ApplicationsTable, CallableComputeProperties, ComputePropertiesLookup, ItemComputeProperties,
+    PackageComputeProperties, PackageStoreComputeProperties,
+};
+use qsc_data_structures::index_map::IndexMap;
+use qsc_fir::fir::{
+    BlockId, ExprId, LocalItemId, PackageId, PackageStore, StmtId, StoreBlockId, StoreExprId,
+    StoreItemId, StoreStmtId,
+};
+
+/// Scaffolding used to build the package store compute properties.
+#[derive(Debug, Default)]
+pub struct PackageStoreScaffolding(IndexMap<PackageId, PackageScaffolding>);
+
+impl ComputePropertiesLookup for PackageStoreScaffolding {
+    fn find_block(&self, id: StoreBlockId) -> Option<&ApplicationsTable> {
+        self.get(id.package)
+            .and_then(|package| package.blocks.get(id.block))
+    }
+
+    fn find_expr(&self, id: StoreExprId) -> Option<&ApplicationsTable> {
+        self.get(id.package)
+            .and_then(|package| package.exprs.get(id.expr))
+    }
+
+    fn find_item(&self, _: StoreItemId) -> Option<&ItemComputeProperties> {
+        panic!("not implemented")
+    }
+
+    fn find_stmt(&self, id: StoreStmtId) -> Option<&ApplicationsTable> {
+        self.get(id.package)
+            .and_then(|package| package.stmts.get(id.stmt))
+    }
+
+    fn get_block(&self, id: StoreBlockId) -> &ApplicationsTable {
+        self.find_block(id)
+            .expect("block compute properties should exist")
+    }
+
+    fn get_expr(&self, id: StoreExprId) -> &ApplicationsTable {
+        self.find_expr(id)
+            .expect("expression compute properties should exist")
+    }
+
+    fn get_item(&self, _: StoreItemId) -> &ItemComputeProperties {
+        panic!("not implemented")
```

Comment:
`notimplemented!()`

Reply:
Did you mean `unimplemented!()`?

Reply:
Updated the code to use `unimplemented!()`.